main.go

```diff
@@ -31,7 +33,11 @@ func main() {
 	models.InitDB()
 
 	// Listen and serve baby
-	fmt.Println("Server starting on http://localhost:8080...")
-	allowedOrigins := handlers.AllowedOrigins([]string{"http://localhost:3000", "http://localhost:5000"})
-	http.ListenAndServe(":8080", handlers.CORS(allowedOrigins)(r))
+	port := os.Getenv("PORT")
+	if port == "" {
+		port = "8080"
+	}
+	fmt.Println("Server starting on http://localhost:" + port + "...")
+	allowedOrigins := handlers.AllowedOrigins([]string{"http://localhost:3000", "http://localhost:5000", "https://frontend-bwkgpgz7aq-uc.a.run.app"})
```

Comment:
Is this identifier permanent? Should we break it out somewhere else?

Reply:
You mean the `https://frontend-bwkgpgz7aq-uc.a.run.app` url? Or what identifier?

Reply:
Yup.

Reply:
Yeah it's persistent